src/components/twilio-voice-basic-call-control/public/twilio-voice-basic-call-control.js

```diff
@@ -85,6 +85,24 @@ class TwilioVoiceBasicCallControl extends HTMLElement {
     this.#showElement(`#${callSid}-unmute`, shouldMute);
   }
 
+  async #handleRemoveParticipant(callSid) {
+    const response = await await fetch(
```

Comment:
I think you wanted use just one `await` here.

Reply:
oops good catch 👍